### DISCUSSION TITLE
구현할 기능 중 질문 있습니다!

### DISCUSSION THREAD

**Discussion by Livenow14** [woowacourse/java-subway-path-precourse#2] [Q&A]
역과 노선, 거리(분)을 저장하는 로직이 초기 설정밖에 없는데, 
이때 역과 노선, 거리(분)은 보장된 값이 들어온다고 생각하면 되나요? 

아니면 보장되지 않은 값이 들어올 수 도 있다고 생각해야 하나요?

**Comment by gracefulBrown** [comment]
요구사항에 제시해드린 내용을 바탕으로 스스로 판단해서 진행을 해주세요!
초기 설정에 대한 요구사항과 사전 등록 정보에 대한 요구사항을 곰곰히 생각해보시면 좋을 것 같아요 :)

**Reply by Livenow14** [reply]
확인했습니다 감사합니다!
